.eslintrc

```diff
@@ -3,7 +3,6 @@
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "plugin:jest/recommended",
-    "plugin:prettier/recommended"
```

Comment:
I'm curious, why remove this?

Reply:
I was finding it a bit annoying to have VSCode showing me formatting errors throughout my code that I'd fix with `yarn format` anyway. https://lifeomic.slack.com/archives/CK65GRH0F/p1587671910471400?thread_ts=1587670515.469700&cid=CK65GRH0F

Reply:
Ah, gotcha. That's an excellent reason.